qt/python/mantidqt/widgets/sliceviewer/test/test_sliceviewer_view.py

```diff
@@ -30,21 +30,22 @@
 
 @start_qapplication
 class SliceViewerViewTest(unittest.TestCase, QtWidgetFinder):
-    def setUp(self):
-        self.histo_ws = CreateMDHistoWorkspace(Dimensionality=3,
-                                               Extents='-3,3,-10,10,-1,1',
-                                               SignalInput=range(100),
-                                               ErrorInput=range(100),
-                                               NumberOfBins='5,5,4',
-                                               Names='Dim1,Dim2,Dim3',
-                                               Units='MomentumTransfer,EnergyTransfer,Angstrom',
-                                               OutputWorkspace='ws_MD_2d')
-        self.hkl_ws = CreateMDWorkspace(Dimensions=3,
-                                        Extents='-10,10,-10,10,-10,10',
-                                        Names='A,B,C',
-                                        Units='r.l.u.,r.l.u.,r.l.u.',
-                                        Frames='HKL,HKL,HKL',
-                                        OutputWorkspace='hkl_ws')
+    @classmethod
+    def setUpClass(cls):
+        cls.histo_ws = CreateMDHistoWorkspace(Dimensionality=3,
+                                              Extents='-3,3,-10,10,-1,1',
+                                              SignalInput=range(100),
+                                              ErrorInput=range(100),
+                                              NumberOfBins='5,5,4',
+                                              Names='Dim1,Dim2,Dim3',
+                                              Units='MomentumTransfer,EnergyTransfer,Angstrom',
+                                              OutputWorkspace='ws_MD_2d')
+        cls.hkl_ws = CreateMDWorkspace(Dimensions=3,
+                                       Extents='-10,10,-9,9,-8,8',
+                                       Names='A,B,C',
+                                       Units='r.l.u.,r.l.u.,r.l.u.',
+                                       Frames='HKL,HKL,HKL',
+                                       OutputWorkspace='hkl_ws')
         expt_info = CreateSampleWorkspace()
         self.hkl_ws.addExperimentInfo(expt_info)
```

Comment:
```suggestion
        cls.hkl_ws.addExperimentInfo(expt_info)
```
This should fix the flake8 warning

Reply:
Thanks, that got introduced when I fixed the merge conflict on github, I was going to fix it but was on another branch so thanks for saving me the time!